AEPEdge.xcodeproj/project.pbxproj

```diff
@@ -1499,13 +1506,15 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 10.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 11.0;
+				MARKETING_VERSION = "";
```

Comment:
Can you remove this empty MARKETING_VERSION? Looks like it was added by accident.